UnitTests/BraintreePaymentFlowTests/MockWebAuthenticationSession.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import AuthenticationServices
+@testable import BraintreePaymentFlow
+
+class MockWebAuthenticationSession: WebAuthenticationSession {
```

Comment:
Similar comment here on punting on duplication - we can likely move `MockWebAuthenticationSession` into `BraintreeTestShared` and share logic to prevent duplication errors.

Reply:
Do we have a JIRA for that work?

Reply:
Not yet. I will make it and link here.